examples/stm32/f1/other/usbh_kbd/Makefile

```diff
@@ -0,0 +1,23 @@
+##
+## This file is part of the libopencm3 project.
+##
+## Copyright (C) 2009 Uwe Hermann <uwe@hermann-uwe.de>
```

Comment:
no, uwe had _nothing_ to do with _this_ file.  Either drop the line altogether, or at least make it correct.